components/microsoft_onedrive/sources/common/base.js

```diff
@@ -1,19 +1,13 @@
-const microsoft_onedrive = require("../../microsoft_onedrive.app");
+const onedrive = require("../../microsoft_onedrive.app");
 const {
   MAX_INITIAL_EVENT_COUNT,
   WEBHOOK_SUBSCRIPTION_RENEWAL_SECONDS,
 } = require("./constants");
-
-function toSingleLineString(multiLineString) {
-  return multiLineString
-    .trim()
-    .replace(/\n/g, " ")
-    .replace(/\s{2,}/g, "");
-}
+const { toSingleLineString } = require("./utils");
 
 module.exports = {
   props: {
-    microsoft_onedrive,
+    onedrive,
```

Comment:
The convention has been to keep this name the same as the app file, if it was renamed to deal with eslint rules I would rather override eslint with a comment and keep the consistent naming.